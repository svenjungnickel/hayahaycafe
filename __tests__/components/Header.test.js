import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../../src/components/Header';

jest.mock('../../src/components/Image');

describe('Header', () => {
    it('renders correctly', () => {
        const headerImage = 'headerImage';
        const title = 'title';
        const subtitle = 'subtitle';

        const component = <Header headerImage={headerImage} title={title} subtitle={subtitle} />;

        expect(renderer.create(component).toJSON()).toMatchSnapshot();
    });
});
