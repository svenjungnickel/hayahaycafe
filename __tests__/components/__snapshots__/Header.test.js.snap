// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="header"
>
  <img
    alt="alt"
    src="src"
  />
  <div
    className="header__content"
  >
    <div
      className="container"
    >
      <div
        className="header__contentInner"
      >
        <h1
          className="display-1"
        >
          title
        </h1>
        <p
          className="lead"
        >
          subtitle
        </p>
      </div>
    </div>
  </div>
</header>
`;
